README.md

```diff
@@ -8,6 +8,48 @@ This library is currently under development and is not yet ready for use.
 The library contains a library of [Nunjucks](https://www.typescriptlang.org)
 components, a CSS stylesheet and a client-side JavaScript library.
 
+## Usage
```

Comment:
I've just put this content here temporarily (while it's fresh in mind). When we have a proper documentation site we can move stuff there (and link to it).